scipy/sparse/compressed.py

```diff
@@ -1276,8 +1281,7 @@ def _process_slice(sl, num):
         if stride != 1:
             raise ValueError('slicing with step != 1 not supported')
         if i0 > i1:
-            raise IndexError(
-                'slicing with start index > stop index is not supported')
+            return 'empty', 'empty'
```

Comment:
This seems like the desired behavior, but is not consistent with older SciPy which did raise an `IndexError` for CSR/CSC in this case. 
e.g. `IndexError: index out of bounds: 0 <= 3 <= 3, 0 <= 2 <= 3, 3 <= 2`

No objections here, but perhaps worth mentioning in the release notes?

Reply:
I think CJ wanted that behavior as you saw above---I'll make a note to adjust the release notes & then you & CJ can check that I've described it properly when I do the backporting.